daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -301,7 +285,7 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
       }
 
     log(LogLevel.Error, "%s", msg)
-    sys.exit(1)
+    sys.exit(64)
```

Comment:
Please add a enum for this value, something like `ExitCode.Usage` probably makes sense to match the name in `sysexits.h`